src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -139,11 +139,17 @@ public int MaxModelValidationErrors
 
         /// <summary>
         /// Gets or sets the flag which causes content negotiation to ignore Accept header 
-        /// when it contains the media type */*. <see langword="false"/> by default.
+        /// when it contains the media type */*. <c>false</c>c> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets the flag to exclude formatters that match only on the object type.
+        /// <c>false</c> by default.
+        /// </summary>
+        public bool ExcludeMatchOnTypeOnly { get; set; }
```

Comment:
I'm having a difficult time parsing this property name. Is there a better name? The property also says nothing about formatters or anything...